modules/payloads/singles/cmd/unix/bind_inetd.rb

```diff
@@ -30,6 +30,9 @@ def initialize(info = {})
           'Payload' => ''
         }
       ))
+    register_options([
+      OptString.new('INETD_PATH', [true, 'The path to inetd', 'inetd'])
+    ])
```

Comment:
The idea of adding paths to all these payloads makes sense, however based on some discussion about user interaction here I think it might be better to have these be `Advanced` option to reduce what the standard user thinks `needs` to be configured for a payload.

```suggestion
    register_advanced_options([
      OptString.new('InetdPath', [true, 'The path to the command executable', 'inetd'])
    ])
```

Advanced options are name in a different casing pattern. 

Based on this suggestion the original `JJS_PATH` options would also need to be converted for consistency.

Reply:
> Consider shifting all these values into `Advanced` options as most users will likely be good with the defaults.  The team is open to leaving them as `standard` options if there is reasoning for why users should always be shown this value. @bcole, what are your thoughts on this?

Since you asked... No strong feelings, but I lean towards making these standard options.

> I think it might be better to have these be `Advanced` option to reduce what the standard user thinks `needs` to be configured for a payload.

Specifying as an option rather than an advanced option was suggested as it is inline with established options for other payloads (reverse_jjs is the only payload which offers a path option IIRC). Although as there's only one instance, it is not a strongly established convention.

The payload path does not seem like a particularly "advanced" option for command payloads. The path absolutely must be configured - but the default value is sane and usually safe to ignore. The command executed by a command payload is an integral part of a command payload.

With one exception, command payloads have never supported specifying the path. Users are unlikely to know that it's even an option. Exposing the path option (instead of hiding it in advanced options) increases the likelihood that users will know that it exists. However, users who want to change the path and are confident editing the source code will likely notice the advanced option when they attempt to edit the payload source, so not a big deal.

Reply:
Thank you for the feedback, both patterns have different merits. We will move forward the new PR and in the future if the advanced option turns out to be too obscure it can be migrated and aliased.

Reply:
@JustAnda7 yes, the goal should be consistency please ensure `jjs_reverset_tcp` is migrated as well.